doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md

```diff
@@ -0,0 +1,251 @@
+
+# Running the Example
+
+The steps that are going to be taken are the following:
+
+* Run the docker compose (*.yml) file
+* Publish Context Information to Orion Context Broker (CB)
+* Create Subscription to notify Cygnus
+* Update Values of an Entity
+* Connect to DB and see the data collected
+
+## Step by step
+
+Remove all docker containers
+
+```console
+docker rm `docker ps -aq`
+```
+
+Remove all docker images
+
+```console
+docker rmi $(docker images -a -q)
+```
+
+To run Cygnus, Orion and mySQL at the same time we need to run the the docker-compose file. We need to change to the directory where we have the docker-compose.yml file and run the following command in the command line prompt
+
+```console
+docker-compose up
+```
+
+To check the images built on your local machine
+
+```console
+
+docker images
+
+REPOSITORY           TAG                 IMAGE ID            CREATED             SIZE
+fiware/orion         latest              5e54ba5d2f2d        2 weeks ago         266MB
+mongo                3.4                 9ad59b0c0624        2 weeks ago         360MB
+mysql                latest              5195076672a7        3 weeks ago         371MB
+fiware/cygnus-ngsi   latest              f8dcb3f89d4a        2 months ago        509MB
+```
+
+To check the current list of running container
+
+```console
+docker ps
+
+CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS                                            NAMES
+7406940f3438        fiware/orion:latest         "/usr/bin/contextBro…"   11 minutes ago      Up 11 minutes       0.0.0.0:1026->1026/tcp                           orion
+c30a8ba06958        fiware/cygnus-ngsi:latest   "/cygnus-entrypoint.…"   11 minutes ago      Up 11 minutes       0.0.0.0:5050->5050/tcp, 0.0.0.0:8081->8081/tcp   cygnus
+187e9b6f9f49        mongo:3.4                   "docker-entrypoint.s…"   11 minutes ago      Up 11 minutes       27017/tcp                                        mongo
+59ed1e7daba8        mysql:latest                "docker-entrypoint.s…"   11 minutes ago      Up 11 minutes       3306/tcp                                         mysql
+```
+
+We see that four containers are running now. Here, we can notice that the Orion Context Broker is running on port 1026. Cygnus-ngsi exposes two ports for management purposes, and 5050 for receiving notifications from Orion. The API must be running on port you configured and 8081 (management port) is the default one. For more info you can check the documentations [here](http://fiware-cygnus.readthedocs.io/en/1.7.1/cygnus-common/installation_and_administration_guide/management_interface_v1/index.html)
```

Comment:
We see that four containers are running now. Here, we can notice that the Orion Context Broker is running on port 1026. Cygnus-ngsi exposes two ports, 8081 for management purposes and 5050 for receiving notifications from Orion. For more info you can check the documentation [here](http://fiware-cygnus.readthedocs.io/en/1.7.1/cygnus-common/installation_and_administration_guide/management_interface_v1/index.html)